_config.yml

```diff
@@ -92,7 +93,7 @@ authors:
   dave:
     name: Dave Goldberg
     display_name: Dave Goldberg
-    email: ""
+    email: "dave.golberg@capitalone.com"
     twitter: "@davidgoldberg"
   davita:
     name: Davita Strong
```

Comment:
No, because then the blog posts she wrote would break:

- https://blog.apiary.io/2015/08/18/Test-Tab-Now-Available-in-Apiary